js/testapps/menu/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "menu",
   "version": "1.0.0",
   "description": "A Genkit test app for understanding a restaurant menu",
-  "main": "lib/index.js",
+  "main": "./lib/src/index.js",
```

Comment:
is this right?

Reply:
Possibly because we are importing some json modules, it creates lib/data and lib/src.